justfile

```diff
@@ -15,40 +15,52 @@ fmt: fmt-nix fmt-haskell fmt-typescript
 check: fmt-nix-check fmt-haskell-check hpack-check fmt-typescript-check
 
 # Deploy the current website
-deploy-website: build-install-script
+deploy-website commit: build-install-script
   #!/usr/bin/env bash
   set -eux
-  if [[ -z $(git status -s) ]]; then
-    COMMIT=$(git rev-parse HEAD)
-    TMP_DIR=$(mktemp --directory)
-    (cd website && npm install && npm run build)
-    ls website/dist/*
-    mv website/dist/* $TMP_DIR
-    git checkout gh-pages
-    cp -rv $TMP_DIR/* .
-    rm -rf $TMP_DIR
-    git add .
-    git commit -m "Website update from commit $COMMIT"
-    echo "Created a new commit. It is not yet pushed."
-  else
-    echo "Working directory is dirty. Please commit, stash or reset before continuing"
-  fi
+
+  TMP_DIR=$(mktemp --directory)
+  DIST_DIR=$TMP_DIR/dist
+  mkdir -p $DIST_DIR
+  cd $TMP_DIR
+  git clone git@github.com:garnix-io/garn
+  cd garn
+  git reset --hard {{ commit }}
+  COMMIT=$(git rev-parse HEAD)
+  (cd website && npm install && npm run build)
```

Comment:
Since this is in the root now and we don't have npm/node in our root flake file this may fail. Should we add them? (This will be so much nicer when we are able to migrate the entire garn repo to use garn :slightly_smiling_face:)

Reply:
Yeah, we should change this, but this was the same before this PR. I didn't want to change too much at once.